analysis/extreme_events_study/README.md

```diff
@@ -0,0 +1,39 @@
+# Extreme Events Study
+
+## How to get dataframes from the output of `extreme_events.py`?
+
+There are multiple dataframes stored as the values of the `dataframe_accumulator` objects in the `dict` output.
+
+```
+import pickle
+import gzip
+
+# Load output
+with gzip.open("path/to/output/file", "rb") as infile:
+    output = pickle.load(infile)
```

Comment:
I'm wondering if you could mention what type of object `output` is? Is this the data frame? Do you have an example of using the data frame? I see below you do e.g. `output["nleps"].values` which looks like that would mean `output` is a dictionary of dictionaries? Is `output` the data frame, or is e.g. `output["nleps"].values` the data frame? Sorry if this question does not make sense...

Reply:
Would it be clearer if I change the first sentence before the code block 

> There are multiple dataframes stored as the values of the dataframe_accumulator objects in the dict output.

to

> The output of the processor is a dictionary of `dataframe_accumulator` objects. The keys in the dictionary are the event characteristics (e.g. nleps is the lepton multiplicity).  Dataframes are stored as `.values` of the items in the dictionary.  An example of getting two dataframes:

Reply:
Thanks, I think this is much more clear.